MVCapplication/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+application.restServiceURL=http://localhost:8080/rest
```

Comment:
To be fully secured. Could you please add to your backlog to use https with self signed certificate
Maybe this link will help
https://hutter.io/2016/02/09/java-create-self-signed-ssl-certificates-for-tomcat/